src/java.desktop/share/classes/com/sun/media/sound/AudioFloatInputStream.java

```diff
@@ -81,8 +82,7 @@ public long getFrameLength() {
         public int read(float[] b, int off, int len) throws IOException {
             if (b == null)
                 throw new NullPointerException();
-            if (off < 0 || len < 0 || len > b.length - off)
-                throw new IndexOutOfBoundsException();
+            Objects.checkFromIndexSize(len, off, b.length);
```

Comment:
Should it be (off, len, b.length)?

Reply:
Well, it seems they are equivalent forms. But yes, `(off, len, b.length)` would be more reasonable.
```
     * <ul>
     *  <li>{@code fromIndex < 0}</li>
     *  <li>{@code size < 0}</li>
     *  <li>{@code fromIndex + size > length}, taking into account integer overflow</li>
     *  <li>{@code length < 0}, which is implied from the former inequalities</li>
     * </ul>
 ```